zeppelin-web/src/components/websocketEvents/websocketMsg.service.js

```diff
@@ -127,6 +127,16 @@ angular.module('zeppelinWebApp').service('websocketMsgSrv', function($rootScope,
       });
     },
 
+    checkpointNotebook: function(noteId, gitCommitMessage) {
+      websocketEvents.sendNewEvent({
+        op: 'CHECKPOINT_NOTEBOOK',
+        data: {
+          noteId: noteId,
+          gitCommitMessage: gitCommitMessage
```

Comment:
Can we use just 'message' or 'commitMessage' instead of 'gitCommitMessage' for the field name?